test/jdk/jdk/nio/zipfs/Utils.java

```diff
@@ -24,24 +24,32 @@
 import java.io.IOException;
 import java.io.OutputStream;
 import java.nio.file.*;
+import java.util.Map;
 import java.util.Random;
 import java.util.jar.JarEntry;
 import java.util.jar.JarOutputStream;
 
+import static java.nio.charset.StandardCharsets.UTF_8;
+
 /**
- * Utility class for zipfs tests.
+ * Utility class for {@code ZipFileSystem} tests.
  */
-
-class Utils {
-    private Utils() { }
+final class Utils {
+    private Utils() {}
 
     /**
-     * Creates a JAR file of the given name with 0 or more named entries.
+     * Creates a JAR file of the given name with 0 or more named entries with
+     * random content.
      *
-     * @return Path to the newly created JAR file
+     * <p>If an existing file of the same name already exists, it is silently
+     * overwritten.
+     *
+     * @param name the file name of the jar file to create in the working directory.
+     * @param entries a list of JAR entries to be populated with random bytes.
+     * @return the absolute path to the newly created JAR file.
      */
     static Path createJarFile(String name, String... entries) throws IOException {
-        Path jarFile = Paths.get("basic.jar");
+        Path jarFile = Paths.get(name);
```

Comment:
In recent times we have replaced `Paths.get(...)` call in the JDK code with `Path.of(...)`. We should do the same here and the other line in this utility class.